gcloud/storage/_helpers.py

```diff
@@ -33,11 +33,6 @@ class _PropertyMixin(object):
     """
 
     @property
-    def connection(self):
```

Comment:
Pleasantly surprised this can be axed. Can't be removed from `Blob/Bucket` until we get `ACL` stuff finished?